Tests/MockoloTestCase.swift

```diff
@@ -4,9 +4,9 @@ import MockoloFramework
 class MockoloTestCase: XCTestCase {
     var srcFilePathsCount = 1
     var mockFilePathsCount = 1
-
+    
```

Comment:
Is this file being intentionally being reformatted?

Reply:
Autoindent seems to have done this. PR #58 has test updates, so it'll be resolved there.